.github/workflows/release.yml

```
@@ -0,0 +1,56 @@
+name: Github + NPM Release
+
+on:
+    push:
+        branches:
+            - 'main'
+
+permissions:
+    contents: write
+    pull-requests: write
+
+jobs:
+    release-please:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: google-github-actions/release-please-action@v3
```

version 4 is available, let's use the latest

https://github.com/google-github-actions/release-please-action

```suggestion
            - uses: google-github-actions/release-please-action@v4
```